index.rst

```diff
@@ -3,11 +3,6 @@
 Symfony Documentation
 =====================
 
-.. toctree::
-    :hidden:
-
-    changelog
```

Comment:
This was intentional as the generated file contains the whole changelog, which is huge and not really useful. But I'm going to remove it for now, and we can decide later if it is relevant to keep it.